incubator/ethereum/values.yaml

```diff
@@ -0,0 +1,41 @@
+# Default values for ethereum.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+imagePullPolicy: IfNotPresent
+
+bootnode:
+  image:
+    repository: ethereum/client-go
+    tag: alltools-v1.7.3
+
+ethstats:
+  image:
+    repository: ethereumex/eth-stats-dashboard
+    tag: v0.0.1
+  webSocketSecret: my-secret-for-connecting-to-ethstats
+  service:
+    type: LoadBalancer
+
+geth:
+  image:
+    repository: ethereum/client-go
+    tag: v1.7.3
+  tx:
+    # transaction nodes
+    replicaCount: 2
+    service:
+      type: ClusterIP
+  miner:
+    # miner nodes
+    replicaCount: 3
+  genesis:
+    # geth genesis block
+    difficulty: '0x0400'
+    gasLimit: '0x8000000'
+    networkId: 98052
+  account:
+    # account to seed and where mined Ether will be deposited
+    publicKey: 0x975aafe74baa1978c6aabd457cf8d90ee3252db0
+    privateKey: 5c539529553e68af419860c617e20f604bd92a6f0c212b3671e0ee1312a33748
```

Comment:
We discussed this. What we'd like to see, prior to merging, is removing any keys by default. So, `publicKey` and `privateKey` would be commented out without a value. Then the NOTES.txt file would look for the existence of those values. If they are not there it would give instructions on supplying them. The deployments could look for those values and only have the deployment yaml if they are supplied.

For example, look at the [apiKey](https://github.com/kubernetes/charts/blob/675fa11bbcf63cab23fe9913b3698853f29dc168/stable/datadog/values.yaml#L60) for datadog. See the [NOTES.txt](https://github.com/kubernetes/charts/blob/675fa11bbcf63cab23fe9913b3698853f29dc168/stable/datadog/templates/NOTES.txt) and the [deployment.yaml](https://github.com/kubernetes/charts/blob/675fa11bbcf63cab23fe9913b3698853f29dc168/stable/datadog/templates/deployment.yaml) templates to see how it's used.

Reply:
Is there a way to do an OR block inside `NOTES.txt`?

I want something like this:
```
{{- if .Values.geth.account.publicKey || .Values.geth.account.privateKey || .Values.geth.account.secret -}}
```

Reply:
I don't think YAML has support for OR operator. I did the logic check based off of if `publicKey` was set.

So, after this change, if you by default go `helm install <this-chart>`. The deployment will fail as the Geth containers can't initialize their accounts (ie. it's broken by default).

Reply:
@jpoon The template yaml files are run through a template system built on go template. Those templates can have `and` and `or` operations. See https://golang.org/pkg/text/template/#hdr-Functions for documentation on it.

Also, there are some CI failures that need to be fixed.

Reply:
Ah, it's a function. Thanks for that :)